src/createClient.spec.ts

```diff
@@ -130,28 +130,93 @@ describe('query', () => {
 })
 
 describe('useQuery', () => {
-  testInEffectScope('responds to parameter changes', async () => {
-    const responseTrue = Symbol('responseTrue')
-    const responseFalse = Symbol('responseFalse')
+  describe('when parameters change', () => {
```

Comment:
these tests all start with "when parameters change..." so I grouped them

Reply:
Nesting is evil lol 👍